README.md

```diff
@@ -11,6 +11,33 @@ Artsy's Design System
 Palette is a collection of primitive, product-agnostic elements that help encapsulate Artsy's look and feel at base level. This project is intended to be used
 across our digital product portfoilo.
 
+Palette is built on top of [Gatsby](https://www.gatsbyjs.org/).
```

Comment:
Since Palette and its docs are separate apps with different requirements we should keep the readme in www and make sure that things work well (add the cd www && yarn install step inside of `yarn docs` script in package.json`)